.github/workflows/build.yml

```diff
@@ -16,20 +16,20 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
-          - 3.0.0
-          - 2.7.2
-          - 2.6.6
-          - 2.5.8
-          - 2.4.10
+          - 3.0
```

Comment:
We will need to add quotes here as otherwise Github Actions interpret this as just a `3` and therefore uses the latest minor version of Ruby 3 for this entry of the Ruby matrix:

https://github.com/nicolastemciuc/cose-ruby/actions/runs/12320821512/job/34390931112#step:3:22

Reply:
Oh, missed that. Seems that now is running against `Ruby 3.0`

<img width="624" alt="image" src="https://github.com/user-attachments/assets/80e49a9c-d12e-4094-9fe0-9cee4ff05a14" />

Thank you!